public/js/project_upload.js

```diff
@@ -0,0 +1,52 @@
+$.noConflict();
+
+jQuery(document).ready(function($) {
+
+	$('#myUpload').on('shown.bs.modal', function () {
+
+		$('#uploadSubmit').prop('disabled', true);
+
+	});
+
+	$("#verifyUrl").click(function() {
+
+    	$.ajax({
+
+    		type : 	"post",
+
+    		url : 	$("#uploadForm").prop('action'),
+
+    		data : 	$("#uploadForm").serialize(),
+
+    		success : 	function(data, status, xhr) {
+    						$("#errors").html("");
+    						$('#uploadSubmit').prop('disabled', false);
+    					},
+
+    		error : 	function (xhr,status,error) {
+    						$("#errors").html("");
+    						$("#uploadSubmit").prop('disabled', true);
+    						for(var key in xhr.responseJSON) {
```

Comment:
The body of a for in should be wrapped in an if statement to filter unwanted properties from the prototype.